src/main/java/com/example/dr_app/service/DoctorService.java

```diff
@@ -0,0 +1,35 @@
+package com.example.dr_app.service;
+
+import com.example.dr_app.model.Doctor;
+import com.example.dr_app.repository.DoctorRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class DoctorService {
```

Comment:
In general, it is a good idea to use Interfaces for services.  
For e.g if your Service is always getting something and setting something you can have an interface with getOne() saveAll() findAll() and you dont need to specify the entity. That can be a generic type. 

Its an andvanced pattern, but worth to explore it. 
https://www.youtube.com/watch?v=Ugf6C-7_Pr4 

For now it's fine to keep it like this.